src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="input input--large"
+      data-qa="big"
+    >
+      <input
+        class="input__field input__field--large"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="input input--small"
+      data-qa="small"
+    >
+      <input
+        class="input__field input__field--small"
+        type="text"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Same as the previous comment, it's not semantically correct to use a `form` element here. Consider changing it to a `div` or other suitable container element.